Grizzlies-SpyDuh/Models/User.cs

```diff
@@ -6,7 +6,18 @@ public class User
         public string Name { get; set; }
         public string Email { get; set; }
         public int AgencyId { get; set; }
-        public virtual Agency Agency { get; set; }
+        public Agency Agency { get; set; }
+
+        public Skill Skill { get; set; }
```

Comment:
Does this reference classes that don't exist in these commits yet? Comment out if not in use to avoid errors.